src/course_bot/presentation.clj

```diff
@@ -219,13 +219,13 @@
       :approve
       (fn [tx {{id :id} :from text :text} {desc :desc}]
         (cond
-          (= text (tr :pres/yes)) (do (talk/send-text token id (tr :pres/teacher-will-check))
```

Comment:
:yes and :no keys (in translation) not used in the code. So they should be removed.

Reply:
Fixed it